tf-modules/archive/cw_sf_event_to_db_records.tf

```diff
@@ -66,6 +80,41 @@ resource "aws_iam_role_policy" "cw_sf_event_to_db_records_lambda_role_policy" {
   policy = data.aws_iam_policy_document.cw_sf_event_to_db_records_lambda.json
 }
 
+
+
+resource "aws_sqs_queue" "cw_sf_event_to_db_records_input_queue" {
+  name = "${var.prefix}-cwSfEventToDbRecordsInputQueue"
+  receive_wait_time_seconds  = 20
+  visibility_timeout_seconds = (aws_lambda_function.cw_sf_event_to_db_records.timeout * 6)
```

Comment:
Very clever, but why this specific multiplier? Don't we just want to ensure that the visibility timeout is at least greater than the lifespan of the Lambda?

Reply:
> To allow your function time to process each batch of records, set the source queue's visibility timeout to at least 6 times the timeout that you configure on your function. The extra time allows for Lambda to retry if your function execution is throttled while your function is processing a previous batch.

https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html